src/EditorFeatures/Core/CommandHandlers/SignatureHelpCommandHandler.cs

```diff
@@ -59,8 +59,8 @@ private bool TryGetController(CommandArgs args, out Controller controller)
         {
             AssertIsForeground();
 
-            // check whether this feature is on.
-            if (!args.SubjectBuffer.GetOption(SignatureHelpOptions.ShowSignatureHelp))
+            // check whether this feature is on, but only if `args` was not InvokeSignatureHelpCommandArgs; in that case always invoke
```

Comment:
Instead of reiterating what the code is doing, can you explain why instead? The question that comes to mind immediately is "yes I know you are doing that, but why?"